services/QuillLMS/db/migrate/20230622125712_add_type_to_auth_credential.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddTypeToAuthCredential < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
You might consider putting this migration and the data migration in a separate PR and deploying that first to cut down on potential errors and/or downtime (All of this logic depends on type being there and populated). Then you can deploy the code and re-run the data migration for the last few records.

Reply:
That's a good point.  I will pull it out in a separate PR.